docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -38,6 +38,13 @@ Then, open the Office application and [sideload your add-in](sideload-an-office-
 
 Add-ins are often cached in Office for Mac, for performance reasons. Normally, the cache is cleared by reloading the add-in. If more than one add-in exists in the same document, the process of automatically clearing the cache on reload might not be reliable.
 
-On a Mac, you can clear the cache manually by deleting the contents of the `~/Library/Containers/com.Microsoft.OsfWebHost/Data/` folder. 
+You can clear the cache using an existing add-in.
```

Comment:
Also, I think the intro blurb to this article is a little confusing. Maybe change to: 
Because add-ins are developed using HTML and JavaScript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on a Mac.
>[!NOTE] 
>You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on a Mac.

Reply:
@mscharlock Thanks! I updated the intro and the date!